sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/EcKeyCryptographyClient.java

```diff
@@ -56,15 +56,13 @@ private KeyPair getKeyPair(JsonWebKey key) {
     }
 
     @Override
-    Mono<EncryptResult> encryptAsync(EncryptionAlgorithm algorithm, byte[] plaintext, byte[] iv,
-                                     byte[] authenticationData, Context context, JsonWebKey key) {
+    Mono<EncryptResult> encryptAsync(EncryptionAlgorithm algorithm, byte[] plaintext, Context context, JsonWebKey key) {
```

Comment:
Since this client has mainly async operations, should it be called `EcKeyCryptographyAsynchClient` and drop all `Async` suffix in method names?

Reply:
internal class, we can rename it later, doesn't matter.